packages/schema/src/types/global/vite.ts

```diff
@@ -1,35 +1,15 @@
 /**
  * Reference: https://github.com/vitejs/vite/blob/main/packages/vite/types/importMeta.d.ts
  */
-
-export interface ViteHot {
-  readonly data: any
-
-  accept (): void
-  accept (cb: (mod: any) => void): void
-  accept (dep: string, cb: (mod: any) => void): void
-  accept (deps: readonly string[], cb: (mods: any[]) => void): void
-
-  dispose (cb: (data: any) => void): void
-  decline (): void
-  invalidate (): void
-
-  on (event: any, cb: (payload: any) => void): void
-  send (event: any, data?: any): void
-}
-
-export interface ViteGlobOptions {
-  as?: string
-  eager?: boolean
-  import?: string
-  query?: string | Record<string, string | number | boolean>
-  exhaustive?: boolean
-}
```

Comment:
For now, could we keep these inlined, but updated to match the latest upstream vite types?

I would be very willing to consider directly referencing or adding vite types to the `nuxt.d.ts` build conditionally, in line with your previous PR - let's discuss at https://github.com/nuxt/nuxt/issues/14131.

Reply:
> updated to match the latest upstream vite types?

Sorry, can you please clarify the benefits ?  This also already has vite as dev dep, so it doesn't introduce any structural change, and I think it doesn't have conflict with `directly referencing or adding vite types to the nuxt.d.ts`, we can always remove this file when it's resolved.

Reply:
The reason these are currently here and not already directly depending on vite's types is that @pi0 was reluctant to depend on vite's types because we may not support all of them, and this way we curate the types that are made available. You can see the context here: https://github.com/nuxt/framework/pull/1439.

Let's discuss a different approach in the linked issue, but for now (in this PR) we should keep the current approach but update the types.

Reply:
Makes sense, I've updated the pr accordingly, thanks!